botocore/docs/method.py

```diff
@@ -207,6 +208,17 @@ def document_model_driven_method(section, method_name, operation_model,
                 request_params_section, operation_model.input_shape,
                 include=include_input, exclude=exclude_input)
 
+
+    # Find idempotent params and append documentation
+    if 'request-params' in section.available_sections:
```

Comment:
Yea I had automatic flake8 on all the time and then I turned that off so I forgot to run it manually.